doc/widgets/wordcloud.md

```diff
@@ -11,9 +11,10 @@ Generates a word cloud from corpus.
 **Outputs**
 
 - Corpus: Documents that match the selection.
-- Word: Selected word that can be used as query in [Concordance](concordance.md).
+- Selected Word: Selected word that can be used as query in [Concordance](concordance.md).
+- Word Counts: Word together withe their weights.
```

Comment:
Word together withe their weights. --> Words and their weights.

Reply:
Fixed.